apps/esbuild-example/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
added an example package that provides a simple build script example for a React application. it is also easier to develop the plugin this way.